blockservice/blockservice.go

```diff
@@ -145,6 +145,6 @@ func (s *BlockService) DeleteBlock(k key.Key) error {
 }
 
 func (s *BlockService) Close() error {
-	log.Debug("blockservice is shutting down...")
+	defer log.Debug("blockservice is shutting down...")
```

Comment:
I was using `defer` to check if the process has truly shut down, i.e. as "blockservice has shut down". Will revert this once I have figured out the `n.Routing`/`n.PeerHost` zombies.